### PR TITLE
Update legacy pages style in order to collapse a bit more to the UIKit

### LINKED ISSUE #22435: [Legacy default theme] - Improve style consistency of legacy pages
#### Is your feature request related to a problem?

Legacy pages have old styles, while new theme received recent update (UIKit), which could be a bit improved in order to win some consistency, even if they are temporary

#### Describe the solution you'd like

We would require to change some styles :

- Buttons
- Forms
- Alerts
- Inputs
- Cards

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Changing some styles on legacy pages : Buttons, Forms, Alerts, Inputs, Cards (I changed old style into UIKit one)
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #22435.
| How to test?  | See issue

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22436)

### CODE REVIEW

File: admin-dev/themes/default/sass/modules/_variables.sass [hunk 1]
```diff
@@ -33,6 +34,10 @@ $gray: lighten(#000, 33.5%)  //#555
 $gray-light: lighten(#000, 60%)  //#999
 $gray-lighter: lighten(#000, 93.5%)  //#eee
 
+// Cards
+$card-border-color: #dbe6e9
+$card-icon-color: #6c868e;
```
Comment (Progi1984) [inline admin-dev/themes/default/sass/modules/_variables.sass]:
```suggestion
$card-icon-color: #6c868e
```

File: admin-dev/themes/default/sass/partials/_buttons.sass [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.btn
```
Comment (Progi1984) [inline admin-dev/themes/default/sass/partials/_buttons.sass:1]: Not the moment to move from SASS to SCSS ?

File: admin-dev/themes/default/sass/partials/_content.sass [hunk 3]
```diff
@@ -51,29 +52,33 @@ body
 			text-overflow: ellipsis
 			white-space: nowrap
 			color: #555
-			height: 32px
+			padding: 10px;
```
Comment (Progi1984) [inline admin-dev/themes/default/sass/partials/_content.sass]:
```suggestion
			padding: 10px
```

**Review by Progi1984 — COMMENTED** [review]
Some feedbacks

File: admin-dev/themes/default/sass/partials/_buttons.sass (hunk 2 above)
Comment (NeOMakinG) [reply admin-dev/themes/default/sass/partials/_buttons.sass:1]: You finally got me...

I'm gonna do it in another PR, when PR touching default will be merged

**Review by atomiix — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @sarahdib & @NeOMakinG

**Pull request merged**